CHANGELOG.md

```diff
@@ -4,7 +4,8 @@
 * None.
 
 ### Enhancements
-* Add support for in-memory Realms.
+* Realm will now use `System.loadLibrary()` first when loading native code on JVM, adding support for 3rd party JVM installers. If this fails, it will fallback to the current method of loading the native library from the JAR file. (Issue [#1105](https://github.com/realm/realm-kotlin/issues/1105)). 
```

Comment:
```suggestion
* Realm will now use `System.loadLibrary()` first when loading native code on JVM, adding support for 3rd party JVM installers. If this fails, it will fallback to the current method of extracting and loading the native library from the JAR file. (Issue [#1105](https://github.com/realm/realm-kotlin/issues/1105)). 
```